app/views/admin/users/_admin_dashboard.html.slim

```diff
@@ -0,0 +1,58 @@
+.row
+  .col-lg-12
+    .page-header
+      h2.center Admin dashboard
+.row
+  .col-md-6.col-lg-6.col-sm-6
+    .panel.panel-default.panel-white.core-box
+      .panel-body.no-padding
+        .partition.partition-grey.padding-20.core-icon.text-center
+          .glyphicon.glyphicon-question-sign.fa-2x
+        .padding-20.core-content
+          h2
+            = Question.count
+          h4 Questions
+      .panel-footer.clearfix.no-padding.text-center
+        a.col-xs-6.padding-10.partition-light-blue href="/admin/questions/new"
+          .glyphicon.glyphicon-plus
+        a.col-xs-6.padding-10.partition-blue href="/admin/questions"
+          .glyphicon.glyphicon-chevron-right
+  .col-md-6.col-lg-6.col-sm-6
+    .panel.panel-default.panel-white.core-box

```

Comment:
all this "panels" could be moved to partial - since on first looks they are similar